crypto/src/lib.rs

```diff
@@ -648,7 +648,7 @@ impl PrivateKey {
     }
 
     /// Key payload
-    fn payload(&self) -> Vec<u8> {
+    pub fn payload(&self) -> Vec<u8> {
```

Comment:
This became private after the merge of #4181. However, `iroha_swarm` relies on this.

@Arjentix, could you comment on it?

Reply:
@Arjentix replied: for this case, `PrivateKey::into_raw` would be more idiomatic. I'll refactor this bit.